linux-setup.sh

```diff
@@ -172,41 +191,37 @@ EOF
     # libncurses-dev and libreadline-dev are needed for readline
     # nginx is used as a devserver proxy that serves static files
     # nodejs is used for various frontendy stuff in webapp, as well as our js
-    #   services. We standardize on version 12 (the latest version suppported
-    #   on appengine standard).
+    #   services. We standardize on version 16.
     # redis is needed to run memorystore on dev
     # libnss3-tools is a pre-req for mkcert, see install_mkcert for details.
     # TODO(benkraft): Pull the version we want from webapp somehow.
-    # curl for various scripts (including setup.sh)
     sudo apt-get install -y git \
-        python-dev \
-        python-mode python-setuptools \
         libfreetype6 libfreetype6-dev libpng-dev libjpeg-dev \
         imagemagick \
         libxslt1-dev \
         libyaml-dev \
         libncurses-dev libreadline-dev \
-        nodejs=12* \
+        nodejs=16* \
```

Comment:
Do we need the `=` at all, after we did the pinning above (at line 124)?  I guess it doesn't hurt, it's just something we'll have to remember to keep in sync going forward.

Reply:
One thing we could do is make the nodejs version an envvar and just use it everywhere we need to in this script.

Reply:
Good catch. We don't need to specify the version in the `apt-get` calls, so I've removed it.